### PR TITLE
qa/suites: drop 'fs' facet, and add 'objectstore' facet where missing

### PR DESCRIPTION
**Pull request by liewegas**
We're currenting testing bluestore on top of both xfs and btrfs, which is silly.
Combine these into a single 'objectstore' face that has bluestore, filestore(xfs),
and filestore(btrfs).

### CODE REVIEW

File: qa/suites/rgw/singleton/fs/xfs.yaml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-overrides:
-  ceph:
-    fs: xfs
-    conf:
-      osd:
-        osd sloppy crc: true
```
Comment (yehudasa) [inline qa/suites/rgw/singleton/fs/xfs.yaml]: @liewegas what's with the `osd sloppy crc`? do we get it from somewhere else?

**Comment by liewegas** [comment]
it's filestore code that opportunistically stores checksums in xattrs.  
meant for qa, not production.  the option is misnamed...

**Comment by yehudasa** [comment]
@liewegas lgtm

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-04-07_19:56:10-rados-wip-sage-testing2---basic-smithi/

**Pull request merged**